api/desecapi/views.py

```diff
@@ -342,20 +340,20 @@ def create(self, request, *args, **kwargs):
             raise exc
 
     def perform_create(self, serializer):
-        serializer.save(domain=self.request.user.domains.get(name=self.kwargs['name']))
+        serializer.save(domain=self.domain)
 
     def update(self, request, *args, **kwargs):
+        partial = kwargs.pop('partial', False)
+        instances = self.get_objects(request, many=isinstance(request.data, list))
+        serializer = self.get_serializer(instances, data=request.data, partial=partial)
+        serializer.is_valid(raise_exception=True)
         with PDNSChangeTracker():
```

Comment:
This could also go inside `perform_update()`.

Reply:
in DRF, that's part of `update`

Reply:
misunderstanding, done